erdpy/README.md

```diff
@@ -1,16 +1,16 @@
 # Description
-Elrond Python Command Line Tools and SDK for interacting with the Elrond Network (in general) and Smart Contracts (in particular). 
+MultiversX Python Command Line Tools and SDK for interacting with the MultiversX Network (in general) and Smart Contracts (in particular). 
```

Comment:
Only now I see that there are 2 readme files. Remove this one, and update the one at the top level?

Reply:
Deleted this file and update the one at the top level.